### ISSUE TITLE
Add only hot corners like a Elementary OS 

### ISSUE THREAD

**Issue by redian23** [nick92/Enso-OS#87]
[Switchboard Desktop Plug](https://github.com/elementary/switchboard-plug-pantheon-shell)

Hot corners it is my favorite option in eOS. 
But Enso have a XFCE (most interesting DE when Pantheon) + gala (WM)
I think it would be better to add H.C. in Enso. 

I try install switchboard-plug-pantheon-shell, but don't have a result.

**Comment by nick92** [comment]
Agreed, it is a nice feature to have, it will need adding into gala as we use a custom build of it, and consign into desktop settings so will look at adding it into the next release

**Comment by redian23** [comment]
Thank you)
I'll wait for the release.
